.circleci/config.yml

```diff
@@ -917,6 +917,45 @@ jobs:
       - run: *gitter_notify_failure
       - run: *gitter_notify_success
 
+  t_ems_compile_ext_ens:
+    docker:
+      - image: circleci/node:10
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Install test dependencies
+          command: |
+            sudo apt-get -qy install lsof
+      - run:
+          name: External Ens compilation
+          command: |
+            export COMPILE_ONLY=1
+            test/externalTests/ens.sh /tmp/workspace/soljson.js
+
+  t_ems_test_ext_ens:
+    docker:
+      - image: circleci/node:10
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Install test dependencies
+          command: |
+            sudo apt-get -qy install lsof
+      - run:
+          name: External Ens tests
+          command: |
+            test/externalTests/ens.sh /tmp/workspace/soljson.js || test/externalTests/ens.sh /tmp/workspace/soljson.js
```

Comment:
Why the duplication?

Reply:
Quoting @chriseth from gitter `We added it because they tended to fail for weird reasons, but maybe it's just more stable now`

Reply:
Ah, I realise now that I forgot to remove the duplication here. Will do!